examples/react-native/README.md

```diff
@@ -1,25 +1,21 @@
-This examples shows how to use `react-instantsearch` along with `react-native`.
+# example-react-native
 
-It was created using `create-react-native-app`.
+_This project was generated with [create-instantsearch-app](https://github.com/algolia/create-instantsearch-app) by [Algolia](https://algolia.com)._
 
-## Clone the example
+This examples shows how to use `react-instantsearch` along with `react-native`.
 
-```
-curl https://codeload.github.com/algolia/react-instantsearch/tar.gz/master | tar -xz --strip=2 react-instantsearch-master/examples/react-native
```

Comment:
is it consistent with the other readmes to no longer have this comment?

Reply:
Indeed, I'll add it back.